build/common.ps1

```diff
@@ -313,7 +313,8 @@ Function New-Package {
 		[string]$ReleaseLabel = "zlocal",
 		[string]$BuildNumber,
 		[switch]$NoPackageAnalysis,
-		[string]$Version
+		[string]$Version,
```

Comment:
It isn't, unfortunately. I made changes in the individual `common.ps1` files as they needed them. I think we might need to look into a long term solution for sharing this build code between our repositories (maybe a submodule) because right now it's unmaintainable.

@xavierdecoster any ideas?